### PR TITLE
Update Changelogs for Boost 1.4.1

### PR DESCRIPTION
**Pull request by thingalon**
This PR just updates the changelogs in prep for a 1.4.1 release. 

Boost 1.4.1 includes Cloud Critical CSS - but keeps it hidden behind a constant.

#### Changes proposed in this Pull Request:
* Prepare changelogs for Boost 1.4.1

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
Please verify that:
- The existing Critical CSS functionality works.
- Everything mentioned in the Jetpack Boost changelog works as described.

### CODE REVIEW

File: projects/plugins/boost/CHANGELOG.md [hunk 1]
```diff
@@ -125,3 +138,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.3.1-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.3.0-beta...v1.3.1-beta
 [1.3.0-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.2.0...v1.3.0-beta
 [1.2.0]: https://github.com/Automattic/jetpack-boost-production/compare/v1.1.0...v1.2.0-beta
+[2.0.0-beta]: https://github.com/Automattic/jetpack-boost-production/compare/v1.4.0...v2.0.0-beta
```
Comment (jeherve) [inline projects/plugins/boost/CHANGELOG.md:141]: I think this should be `1.4.1-beta`, and should be above the other links?

File: projects/plugins/boost/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+- Added internal support for Cloud Critical CSS.
+- Added My Jetpack page.
+- Added suggested steps when Critical CSS fails due to page errors.
+
+### Changed
+- Critical CSS: Exclude animation keyframes from Critical CSS Generation.
+- Deferred JS: Attribute to exclude scripts from deferral now allows quotes.
+- General: Boost is now compatible with WordPress 5.9.
+- Tidy up Critical CSS class structure.
+- Updated Dashboard heading with new Horizontal logo.
```
Comment (jeherve) [inline projects/plugins/boost/CHANGELOG.md:19]: There is a bit of a mix of present and past tense in here, and some entries with a prefix (General, Critical CSS, Deferred JS) while others do not have one. Maybe we could standardize this?

File: projects/plugins/boost/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jetpack-boost",
-	"version": "2.0.0-alpha",
+	"version": "2.0.0-beta",
```
Comment (jeherve) [inline projects/plugins/boost/package.json:3]: This doesn't seem right, since we're releasing 1.4.1-beta.

**Review by jeherve — CHANGES_REQUESTED** [review]

File: projects/plugins/boost/CHANGELOG.md (hunk 1 above)
Comment (thingalon) [reply projects/plugins/boost/CHANGELOG.md]: Yes, it should be. Fixed in 8fd2204189. Thank you

File: projects/plugins/boost/CHANGELOG.md (hunk 2 above)
Comment (thingalon) [reply projects/plugins/boost/CHANGELOG.md]: Fixed in f5f3d43603.

File: projects/plugins/boost/package.json (hunk 3 above)
Comment (thingalon) [reply projects/plugins/boost/package.json]: Thanks; I missed this one. Fixed it, and `composer.json` in 2fe9295108.

File: projects/plugins/boost/readme.txt [hunk 4]
```diff
@@ -165,25 +165,19 @@ If you run into compatibility issues, please do let us know. You can drop us a l
...
+### 1.4.1-beta - 2022-03-08
 #### Added
-- Critical CSS: Add extra information to "fetch" errors when generating Critical CSS.
-- Critical CSS: Added explanation for mod-security HTTP 418 errors.
-- Critical CSS: Added stats tracking for generation outcomes.
-- Critical CSS: Added step-by-step instructions for Advanced Recommendations.
-- Critical CSS: More descriptive error message if critical css is failing because of x-frame-options deny config.
-- Speed Scores: Added "without Boost" speed score indicator.
+- Added internal support for Cloud Critical CSS.
+- Added My Jetpack page.
```
Comment (eAdnan007) [inline projects/plugins/boost/readme.txt:171]: This is not true. My jetpack was added in 1.4.0 but it is kept hidden unless a constant is specified.

File: projects/plugins/boost/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.1-beta] - 2022-03-08
+### Added
+- Critical CSS: Added internal support for Cloud Critical CSS.
+- Critical CSS: Added suggested steps when Critical CSS fails due to page errors.
+- General: Added "My Jetpack" page.
```
Comment (eAdnan007) [inline projects/plugins/boost/CHANGELOG.md:12]: My Jetpack was added(but not activated) on 1.4.0

File: projects/plugins/boost/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
```
Comment (eAdnan007) [inline projects/plugins/boost/CHANGELOG.md:14]: Should we mention this issue: https://github.com/Automattic/jetpack/issues/21585

**Review by haqadn — CHANGES_REQUESTED** [review]
Some minor points to check

**Review by haqadn — CHANGES_REQUESTED** [review]
Some minor points to look at

File: projects/plugins/boost/CHANGELOG.md (hunk 6 above)
Comment (thingalon) [reply projects/plugins/boost/CHANGELOG.md:13]: Sure. Added in e22bc6b31c.

File: projects/plugins/boost/CHANGELOG.md (hunk 5 above)
Comment (thingalon) [reply projects/plugins/boost/CHANGELOG.md]: Yep. e22bc6b31c

File: projects/plugins/boost/readme.txt (hunk 4 above)
Comment (thingalon) [reply projects/plugins/boost/readme.txt]: Thanks Adnan e22bc6b31c

**Comment by thingalon** [comment]
image

Well, now this has circled the drain for a while I'll have to throw this branch away and try again because... well... yeah.

**Pull request closed without merging**